src/components/graph/Graph.jsx

```diff
@@ -101,6 +101,10 @@ import { merge, throwErr } from "../../utils";
  *      window.alert(`Node ${nodeId} moved to new position x= ${x} y= ${y}`);
  * };
  *
+ * const onZoomChange = function(newZoom) {
```

Comment:
Please add some jsdoc on this `onZoomChange` (example with the new API in mind):

```javascript
/**
 * This function is triggered whenever there's a change in the zoom value.
 * @param {number} prevZoom the previous zoom value
 * @param {number} currZoom the current zoom value
 */
const onZoomChange = function(newZoom) {
//...
```